cfg/config.yaml

```diff
@@ -131,14 +140,17 @@ node:
       - "hyperkube kube-proxy"
       - "proxy"
     confs:
+      - /var/snap/microk8s/current/args/kube-proxy
```

Comment:
Please let's put this one as the last option in the list (they get checked in order)